test/TesterInternal/OrleansRuntime/Streams/ObjectPoolTests.cs

```diff
@@ -72,5 +75,38 @@ public void Alloc10Free1Test()
             // only 1 at a time was ever allocated, but pool was pre-populated with 10, so max allocated should be 10
             Assert.AreEqual(10, accumulator.MaxAllocated);
         }
+
+        [Fact, TestCategory("BVT"), TestCategory("Streaming")]
+        public void ReuseResourceTest()
+        {
+            var accumulator = new Accumulator();
+            IObjectPool<TestPooledResource> pool = new ObjectPool<TestPooledResource>(() => new TestPooledResource(accumulator), 10);
+
```

Comment:
especially in this test, I would use a constant for the number 10, and use numbers relative to that constant to make it clear why you loop to 10 or 20 in your tests